examples/complete/android/demo/src/main/java/com/amazon/styledictionaryexample/models/Property.kt

```diff
@@ -1,35 +1,28 @@
 /**
  * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
  *
+ *
```

Comment:
I think these additional comment lines got added on accident?

Reply:
Yeah! Looks like after automatic conversion to Kotlin. Removed